web/src/lib/components/shared-components/side-bar/server-status.svelte

```diff
@@ -15,11 +15,14 @@
 
   let isOpen = false;
 
-  $: version = $serverVersion ? `v${$serverVersion.major}.${$serverVersion.minor}.${$serverVersion.patch}` : null;
-
   let info: ServerAboutResponseDto;
   let versions: ServerVersionHistoryResponseDto[] = [];
 
+  $: isMain = info?.sourceRef === 'main';
```

Comment:
This is just build metadata and it could be from an entirely different source repository. I think we should have a repositoryUrl AND sourceRef check for the warning.